src/main/java/uk/gov/hmcts/ccd/domain/service/util/TransactionHelper.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.ccd.domain.service.util;
+
+import org.springframework.stereotype.Component;
```

Comment:
is this class a leftover? can be removed?

Reply:
yeah.. removed.